miniapp/USERGUIDE.md

```diff
@@ -93,7 +93,17 @@ CoroutineScope(Dispatchers.Default).launch {
 
 **Note:** This SDK uses `suspend` functions, so you should use [Kotlin Coroutines](https://kotlinlang.org/docs/reference/coroutines-overview.html) when calling the functions. These examples use `Dispatchers.Default`, but you can use whichever `CoroutineContext` and `CouroutineScope` that is appropriate for your App.
 
-### #4 Create and display a Mini App
+### #4 Implement the MiniAppMessageBridge
+
+The `MiniAppMessageBridge` is used for passing messages between the Mini App (JavaScript) and the Host App (your native iOS App) and vice versa. Your App must provide the implementation for these functions and pass this implementation to the `MiniApp#create` function.
```

Comment:
Maybe replace `native iOS App` text with android one 😄 ?

Reply:
😣  That's what happens when I try to save time by copying and pasting from iOS 😆